src/registration_and_login/LoginForm.js

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+import validator from 'validator';
+import { isEmail, isLength } from 'validator';
+import { useNavigate } from 'react-router-dom';
+
+const LoginForm = () => {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [isValidEmail, setIsValidEmail] = useState(true);
+  const [error, setError] = useState('');

```

Comment:
This component is making a request which can take some time so it should also render a progress indicator. So you should also use  `const [isLoading, setIsLoading] = useState(false)` to keep track of that.

Reply:
Solved. {isLoading && <h1>loading</h1>} added under form

Reply:
I don't see the change. Have you forgot to push your latest changes?